src/objects/zcl_abapgit_object_fugr.clas.abap

```diff
@@ -534,9 +537,29 @@ CLASS zcl_abapgit_object_fugr IMPLEMENTATION.
       zcx_abapgit_exception=>raise_t100( ).
     ENDIF.
 
-* The result can also contain function which are lowercase.
+    SELECT * FROM enlfdir
+      INTO TABLE lt_enlfdir
+      WHERE area = ms_item-obj_name
+        AND active = abap_true
+      ORDER BY funcname.
+
+    IF sy-subrc <> 0.
```

Comment:
I think it's possible to have a FUGR without function modules, for example right after you create it

Reply:
You're right, I just tested it. Removed message.